.github/dependabot.yml

```diff
@@ -5,12 +5,11 @@
 
 version: 2
 updates:
-  - package-ecosystem: "npm"
-    directory: "ui/"
+  - package-ecosystem: 'npm'
+    directory: 'ui/'
     schedule:
-      interval: "weekly"
-      day: "thursday"
-    open-pull-requests-limit: 10
```

Comment:
Sure if you can explain why it was set in the first place 😜 
IMHO limiting the number of PRs is an anti-pattern. It shows that we have too many out of date dependencies and do not regularly update them. Also, if there are PRs created that we cannot merge because they require further investigation (like the npm react ones, or due to tests failing), we will not receive any more PRs for this dependency type, which in turn may prevent us from noticing and closing a critical CVE.